### ISSUE TITLE
Add UUSD to Gnosis bridge's token list

### ISSUE THREAD

**Issue by zugdev** [pavlovcik/uad-ui#975]
At https://bridge.gnosischain.com UBQ is listed as a bridgeable token, we should make UUSD appear on that list as well.

![image](https://github.com/user-attachments/assets/e5bd38b4-0509-45fd-aabe-2f0e3153a9c6)

![image](https://github.com/user-attachments/assets/d063278d-8526-492e-ac7f-8ee3ba6da4e7)

**Comment by 0x4007** [comment]
Its unclear to me how to add there but UBQ has been around for four years, unlike UUSD. Perhaps UBQ listed somewhere that Gnosis uses.

**Comment by zugdev** [comment]
> Its unclear to me how to add there but UBQ has been around for four years, unlike UUSD. Perhaps UBQ listed somewhere that Gnosis uses.

We can reach out to their team to find that out. If we prove we are a real project they should add that manually. The same should eventually be done to add the token to Rabby, DefiLlama and other wallets.

**Comment by molecula451** [comment]
I don't think this bridge is open-source to make a contribution and add the token to the metadata most likely they only have tokens they like to make it look like an "official bridge" without aggregating others ERC20

**Comment by 0x4007** [comment]
https://discord.com/invite/gnosis

I also just ran a `wget -r` to surf the source code right now.

api.covalenthq.com https://github.com/covalenthq ?

Just need to find where they source their token list from.

**Comment by rndquu** [comment]
Closing in favor of https://github.com/ubiquity/ubiquity-dollar/issues/984
